test/windows-bootstrap.el

```diff
@@ -30,7 +30,7 @@
                                          user-emacs-directory))
 
 (let* ((package-archives '(("melpa" . "https://melpa.org/packages/")
-                           ("gnu" . "https://elpa.gnu.org/packages/")))
+                           ("gnu" . "http://elpa.gnu.org/packages/")))
```

Comment:
This has security and privacy implications. Are you sure there is no other way?

Reply:
Sometimes it works; sometimes it does not work. 😕 I have attempted to fix for all other my packages and my config but no luck. Feel free to correct, and I indeed want a better solution. Otherwise, I don't think there is other way to get around this issue.